SPECS/python-rich/python-rich.spec

```diff
@@ -51,6 +59,7 @@ code, tracebacks, and more — out of the box.
 
 %check
 # add below to make sure initial build will catch runtime import errors
+pip3 install iniconfig pygments
```

Comment:
I'm confused. If we're grabbing the latest `pygments` here, what is the 0001 patch for?

Reply:
This was here for testing, I will remove. Good catch!

Reply:
removed